include/boost/wave/util/cpp_macromap_predef.hpp

```diff
@@ -240,6 +240,23 @@ namespace util {
         }
 #endif
 
+#if BOOST_WAVE_SUPPORT_CPP2A != 0
+    // C++20 mode
+        static_macros const& static_data_cpp2a(std::size_t i) const
+        {
+        static static_macros data[] = {
+                { "__STDC__", T_INTLIT, "1" },
+                { "__cplusplus", T_INTLIT, "202002L" },
+                { "__STDC_VERSION__", T_INTLIT, "199901L" },
+                { "__STDC_HOSTED__", T_INTLIT, "0" },
+                { "__WAVE_HAS_VARIADICS__", T_INTLIT, "1" },
```

Comment:
Should we define a `{ "__WAVE_HAS_VA_OPT__", T_INTLIT, "1" },` to enable for user code to detect support for it? I'm on the fence here whether this is necessary...

Reply:
I'm inclined to leave it out :) but maybe @eldiener would have an opinion on the subject.

Reply:
If there are no objections, can we leave this out for now until someone has an application?